django/utils/choices.py

```diff
@@ -0,0 +1,24 @@
+import collections.abc
+
+from django.db.models.enums import ChoicesMeta
+
+
+def normalize_field_choices(value):
```

Comment:
There seems to be missing a new test file for this new module, perhaps? Following the current pattern, it seems like we should add a `tests/utils_tests/test_choices.py` with a testcase for this new function.

Reply:
Yeah, so I'd relied on all of the existing tests implicitly showing that this works, but something explicit does make sense - just hadn't got around to it yet 😅 Incoming...